src/diagnostics.rs

```diff
@@ -219,10 +221,27 @@ pub fn report_error<'tcx, 'mir>(
                 }
                 helps
             },
-            TreeBorrowsUb { .. } => {
-                let helps = vec![
+            TreeBorrowsUb { trigger_history, faulty_history, .. } => {
+                let url = "https://perso.crans.org/vanille/treebor";
```

Comment:
I agree that we should link people to something helpful, and also that your write-up is helpful, but I'm not excited about linking someone to a site that's not managed by the Rust project.

Reply:
Very understandable. I'll remove that.

Reply:
FWIW the readme still links there.

We could do what we did with SB, and have a page in https://github.com/rust-lang/unsafe-code-guidelines/tree/master/wip and link there. For now that page would basically just link to Neven's blog, but in the future it could contain its own concise description of the rules (like we do for SB).